Templates/TraditionalBridge/InterfaceDispatcherData.tpl

```diff
@@ -55,15 +55,12 @@ namespace <NAMESPACE>
 		;;; Initialize all data object metadata
 		;;; <summary>
 		private method initMetaData, void
-		record
-			dataObj, @DataObjectMetadataBase
 		proc
 			<STRUCTURE_LOOP>
-			xcall Meta<StructureNoplural>(dataObj)
-			DataObjectMetadataBase.ForceLookupType("<StructureNoplural>", dataObj)
+			DataObjectMetadataBase.LookupType("<StructureNoplural>")
```

Comment:
whats the purpose of this change? This will paper over any missing dataobjects that havent been registered properly.

Reply:
Undid this change in 670b7aa70995b95016f2b252d1c26749a9eafd25